prime-router/settings/organizations.yml

```diff
@@ -2418,6 +2418,14 @@
   filters:
     - topic: covid-19
       jurisdictionalFilter: [ "matches(ordering_facility_state, IG)" ]
+    - topic: full-elr
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state.exists() and Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state = 'IG') or (Bundle.entry.resource.ofType(Patient).address.exists() and Bundle.entry.resource.ofType(Patient).address.state = 'IG')"
```

Comment:
I think this can use the same setup as the CO filters to make them more readable with %patientState, etc.

Reply:
Yup, looks a lot better after doing that